src/SmartComponents/Drift/DriftCard.js

```diff
@@ -136,7 +135,7 @@ const DriftCard = () => {
                                                             dataListCells={[
                                                                 <React.Fragment key={index}>
                                                                     <DataListCell key={`title-${index}`}
-                                                                        wrapModifier={DataListWrapModifier.truncate}>
+                                                                        wrapModifier={'truncate'}>
```

Comment:
We still should use the `DataListWrapModifier` constant for this we'll need to add an alias for it in the `tsc-transforms` here: https://github.com/RedHatInsights/frontend-components/blob/master/packages/tsc-transform-imports/src/index.ts#L85

Reply:
I understand that we would like to add more stuff to the cache and use it everywhere we need. But this is the only place in red hat insights org where we use DataListWrapModifier. I think it is safer and faster to change it to a string
![image](https://github.com/RedHatInsights/insights-dashboard/assets/62722417/ad0eb0de-d447-4a7a-8fda-f0a50940a831)

Reply:
You just need to add 
```js
DataListWrapModifier: getModuleExplicitLocation(CORE_DIRECTORIES, 'components/DataList'),
``` 

to [packages/tsc-transform-imports/src/index.ts#L85](https://github.com/RedHatInsights/frontend-components/blob/master/packages/tsc-transform-imports/src/index.ts#L85) and bump the package here after it is released.

Reply:
If other apps then also use the `DataListWrapModifier` they won't encounter this issue and worry about replacing it again. It's not only about updating a single app and just get it working, but do it correctly as well and not have to go over everything again.

Reply:
Ok, understand. I will prepare a PR